assets/sass/common/shared/o-forms/_o-form.scss

```diff
@@ -715,7 +715,7 @@ only screen and (min-resolution: 2dppx) {
   padding-left: 18px;
   position: relative;
   display: none;
-  word-break: break-all;
```

Comment:
`o-form.scss` was forked from courage a while ago and looks like courage doesn't have `work-break` nor `overflow-wrap` for `.o-form-explain.o-form-input-error`. (`courage/assets/sass/o-forms/_all.scss`).
Curious courage today has same problem?
(trying to align with courage if possible so that could simplify to the upgrade later (reuses SASS from courage)

Reply:
Oops, sorry, I ended up rebasing off of master and merging this yesterday (wasn't sure when you were going to be back from PTO and didn't want to miss the End of Week considering the holidays) - https://github.com/okta/okta-signin-widget/pull/1526

Reply:
nvm but would be great if you could help to take a look as follow up.